### PR TITLE
Add three test cases for Evolution

### PR DESCRIPTION
**Pull request by QingmingSu**
evolution_smoke: Evolution wizard validation
evolution_mail_imap: Case #1503768: send and receive email via IMAP
evolution_mail_ews: Case #1503965: Setup MS Exchange account

### CODE REVIEW

File: tests/x11regressions/evolution/evolution_mail_ews.pm [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    send_key "ret";
+    if (check_screen "evolution_mail-init-window", 30) {
+        send_key "super-up";
+    }
+    assert_screen "evolution_mail-max-window", 30;
+
+    # Make all existing mails as read
+    assert_screen "evolution_mail-online", 60;
+    assert_and_click "evolution_mail-inbox";
+    sleep 1;
```
Comment (coolo) [inline tests/x11regressions/evolution/evolution_mail_ews.pm]: what's the point of this sleep? The assert_screen will wait anyway.

**Comment by coolo** [comment]
please remove all timeouts on assert_screen that are <= 31.

**Comment by QingmingSu** [comment]
Yeah, I see. The "sleep" before assert_screen isn't required. And all timeouts on assert_screen that are <=31 should be removed since the default timeout is 30s. 
Seems the timeout on send_key should be replaced with wait_screen_change.
I will update the tests and submit a new pull request.

**Comment by QingmingSu** [comment]
I will submit new pull request when the tests updated and the Travis-CI check passed.

**Comment by coolo** [comment]
there is no timeout argument to send_key - but that's a common misunderstanding ;)

**Pull request closed without merging**